docs/docsite/rst/conf.py

```diff
@@ -382,48 +382,49 @@
+# Then update each entry in this mapping to have this format:
 #   name: ('http://UPSTREAM_URL', (None, 'path/to/local/cache.inv'))
 #
 # The update script depends on this format so deviating from this (for instance, adding a third
 # location for the mapping to live) will confuse it.
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/2/', None, None),
-    'python3': ('https://docs.python.org/3/', None, None),
-    'jinja2': ('http://jinja.palletsprojects.com/', None, None),
-    'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', None, None),
-    'ansible_9': ('https://docs.ansible.com/ansible/9/', None, None),
+    'python': ('https://docs.python.org/2/', (None, None)),
+    'python3': ('https://docs.python.org/3/', (None, None)),
+    'jinja2': ('http://jinja.palletsprojects.com/', (None, None)),
+    'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', (None, None)),
+    'ansible_9': ('https://docs.ansible.com/ansible/9/', (None, None)),
 } if tags.has('all') else {
-    'python': ('https://docs.python.org/2/', None, None),
-    'python3': ('https://docs.python.org/3/', None, None),
-    'jinja2': ('http://jinja.palletsprojects.com/', None, None),
-    'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', None, None),
-    'ansible_9': ('https://docs.ansible.com/ansible/9/', None, None),
+    'python': ('https://docs.python.org/2/', (None, None)),
+    'python3': ('https://docs.python.org/3/', (None, None)),
+    'jinja2': ('http://jinja.palletsprojects.com/', (None, None)),
+    'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', (None, None)),
+    'ansible_9': ('https://docs.ansible.com/ansible/9/', (None, None)),
 } if tags.has('core_lang') else {
     'python': ('https://docs.python.org/2/', (None, None)),
     'python3': ('https://docs.python.org/3/', (None, None)),
     'jinja2': ('http://jinja.palletsprojects.com/', (None, None)),
     'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', (None, None)),
     'ansible_9': ('https://docs.ansible.com/ansible/9/', (None, None)),
 } if tags.has('core') else {
-    'python': ('https://docs.python.org/2/', (None, '../python2.inv')),
-    'python3': ('https://docs.python.org/3/', (None, '../python3.inv')),
-    'jinja2': ('http://jinja.palletsprojects.com/', (None, '../jinja2.inv')),
-    'ansible_2_10': ('https://docs.ansible.com/ansible/2.10/', (None, '../ansible_2_10.inv')),
-    'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', (None, '../ansible_2_9.inv')),
-    'ansible_2_8': ('https://docs.ansible.com/ansible/2.8/', (None, '../ansible_2_8.inv')),
-    'ansible_2_7': ('https://docs.ansible.com/ansible/2.7/', (None, '../ansible_2_7.inv')),
-    'ansible_2_6': ('https://docs.ansible.com/ansible/2.6/', (None, '../ansible_2_6.inv')),
-    'ansible_2_5': ('https://docs.ansible.com/ansible/2.5/', (None, '../ansible_2_5.inv')),
+    'python': ('https://docs.python.org/2/', (None, None)),
```

Comment:
```suggestion
    'python': ('https://docs.python.org/2/', (None)),
```

Reply:
Should all be a single `None` to prevent Sphinx from doing the same thing twice as @webknjaz mentioned AFAIK